final/challenges.js

```diff
@@ -0,0 +1,133 @@
+/*
+YOUR 3 CHALLENGES
+Change the game to follow these rules:
+
+1. A player looses his ENTIRE score when he rolls two 6 in a row. After that, it's the next player's turn. (Hint: Always save the previous dice roll in a separate variable)
+2. Add an input field to the HTML where players can set the winning score, so that they can change the predefined score of 100. (Hint: you can read that value with the .value property in JavaScript. This is a good oportunity to use google to figure this out :)
+3. Add another dice to the game, so that there are two dices now. The player looses his current score when one of them is a 1. (Hint: you will need CSS to position the second dice, so take a look at the CSS code for the first one.)
+*/
+
+var scores, roundScore, activePlayer, gamePlaying;
+
```

Comment:
Some of these variables are being treated as arrays elsewhere in the code.
In order to help your IDE to better inform you of errors (if using JS 5) then you should declare
the variable as follows:
var purpleMonkey = new Array();
OR
var purpleMonkey = [];
N.B From JS2015 onawards you would use the following syntax:
let purpleMonkey = new Array();